DNN Platform/Library/Common/Internal/IGlobals.cs

```diff
@@ -28,7 +31,7 @@ public interface IGlobals
         /// </summary>
         /// <param name="fileNamePath">The folder the absolute path.</param>
         /// <param name="portalId">Portal Id.</param>
-        /// <returns></returns>
+        /// <returns>The subforlder path.</returns>

```

Comment:
Small spelling mistake - "subforlder" to "subfolder"

Other than that, looks good. Nice job

Reply:
Thank you, corrected in https://github.com/dnnsoftware/Dnn.Platform/pull/4243/commits/05d426fdfeb31929b2333fa20ebab36f4ea563f0